install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3340,6 +3340,10 @@ cgroup:
     #     memory: 128Mi
   # -- Configure cgroup root where cgroup2 filesystem is mounted on the host (see also: `cgroup.autoMount`)
   hostRoot: /run/cilium/cgroupv2
+# -- Configure sysctl override described in #20072.
+sysctlfix:
+  # -- Enable the sysctl override. When enabled, the init container will mount the /proc of the host so that the `sysctlfix` utility can execute.
+  enabled: true
```

Comment:
When would you ever disable this?

Reply:
When people don't want to let Cilium to mount `/proc`. We also have a similar knob for cgroup.autoMount.